### ISSUE TITLE
Memory leak in Julia 1.0

### ISSUE THREAD

**Issue by longqian95** [JuliaPy/PyCall.jl#573]
```
using PyCall
@pyimport cv2
for i=1:100; t=cv2.GaussianBlur(zeros(1000,1000),(3,3),1); end
```
The last line will cause memory leak. `GC.gc()` helps nothing. But in Julia0.6, there is totally no problem.

`Pkg.installed()["PyCall"]`: v1.18.4
`PyCall.pyversion`: v2.7.15
`cv2.__version__`: 2.4.13.5

**Comment by pevnak** [comment]
I think that I am having the same problem in Julia 0.7.  Any progress on this side?

**Comment by tkf** [comment]
It's probably already fixed in master by #558.

**Comment by stevengj** [comment]
Closing as likely fixed by #558.
